### PR TITLE
Introduce Network Policies feature

### PR DESCRIPTION
**Pull request by wchrapka**
**Description**

Adds the sets of Network Policy views to Network and Discovery menu

**Related issue(s)**

Solves #948

### CODE REVIEW

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -1403,6 +1404,28 @@ namespaces:
     total-memory-limit: Define constraints that limit total memory consumption in your Namespace.
     usage-of-memory-limits: This Namespace uses {{valueText}} of {{maxText}} its memory limits.
     usage-of-memory-requests: This Namespace uses {{valueText}} of {{maxText}} its memory requests.
+network-policies:
+  description: <0>Network Policy</0> allows you to specify how a pod is allowed to communicate with various network entities.
```
Comment (grego952) [inline core/src/i18n/en.yaml:1408]:
```suggestion
  description: <0>Network Policy</0> allows you to control how a Pod communicates with different network entities.
```

**Review by grego952 — APPROVED** [review]

File: core-ui/src/components/Predefined/Details/NetworkPolicy/LabelSelector.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import { useTranslation } from 'react-i18next';
+
+export const LabelSelector = ({ selector, title }) => {
+  const { t, i18n } = useTranslation();
+  if (selector?.matchLabels)
+    return (
+      <LayoutPanel
+        className="fd-margin--md"
+        key="policy-types"
+        data-testid="hpa-spec-ref"
```
Comment (Wawrzyn321) [inline core-ui/src/components/Predefined/Details/NetworkPolicy/LabelSelector.js:15]:
```suggestion
```

File: core-ui/src/components/Predefined/Details/NetworkPolicy/Peers.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+              <LayoutPanelRow
+                name={t('network-policies.headers.exceptions')}
+                value={
+                  peer.ipBlock.except ? (
+                    <Tokens tokens={peer.ipBlock.except} />
+                  ) : (
+                    EMPTY_TEXT_PLACEHOLDER
+                  )
+                }
+              />
```
Comment (Wawrzyn321) [inline core-ui/src/components/Predefined/Details/NetworkPolicy/Peers.js:37]:
```suggestion
                value={<Tokens tokens={peer.ipBlock?.except} />}
```

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
Would you kindly list also Pods affected by a policy?

**Review by Wawrzyn321 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**